ext/standaloneusers/ang/crmResetPassword/crmResetPassword.html

```diff
@@ -3,7 +3,7 @@
   <form name="requestLink" crm-ui-id-scope
     ng-if="!$ctrl.formSubmitted && !$ctrl.token">
 
-    <h1>Request Password Reset Link</h1>
+    <h1>{{ts('Request Password Reset Link')}}</h1>
```

Comment:
I think this needs to be `{{:: ts('Request Password Reset Link') }}` right @colemanw ?

Reply:
Yes the `::` is more efficient as it only runs the ts() function once instead of once-per-digest-loop.

Reply:
Indeed. Updated.